src/lib/libwebaudio.js

```diff
@@ -100,9 +95,15 @@ var LibraryWebAudio = {
 #endif
 #endif
 
+    // AUDIO_CONTEXT_RENDER_SIZE_DEFAULT and AUDIO_CONTEXT_RENDER_SIZE_HARDWARE
+    // into their AudioContextRenderSizeCategory enum, or take the positive int. 
+    function readRenderSizeHint(val) {
+        return (val == 0) ? "default" : ((val < 0) ? "hardware" : val);
```

Comment:
I think it is better to be clearer and have a couple of extra lines here. Nested ternary expressions are hard to read.


```suggestion
        if (val == 0) return "default";
        if (val == -1) return "hardware";
        return val;
```